README.md

```diff
@@ -1,8 +1,262 @@
-# RFC
+# Amethyst RFCs - [Active RFC List TBD]
```

Comment:
We should just link to the open PRs, but I don't think that needs to go into the headline.

Reply:
Ok, will leave the link out until I have the bot set up

Reply:
Update: Provide three links:

* discussing RFCs (should link to all open PRs here)
* accepted RFCs (can link to the file tree)
* RFCs waiting for implementation (issues with `label="RFC accepted" -label="status: working"`

Reply:
OK, I've added these notes in. Actual links will have to wait until I have the bot running.